index.html

```diff
@@ -35,6 +35,11 @@
 	<form class="form-popup" id="form_position_details">
 		<h1>Position Details</h1>
 		<hr>
+		<div class="row">
+			<label for="details_name" class="left">Description</label>
```

Comment:
Most of the js part calls this param "name", it might be more consistent to call it "Name" here as well, or just description in the js part too.